pkg/controller/master/upgrade/upgrade_controller.go

```diff
@@ -968,3 +974,45 @@ func (h *upgradeHandler) startVM(ctx context.Context, vm *kubevirtv1.VirtualMach
 		Do(ctx)
 	return res.Error()
 }
+
+// checkLogReadyCondition times out LogReady condition, thus moving on to the next step if upgradeLogCondLogReadyTimeout
+// time has passed since UpgradeLog's logging infrastructure setup started
+func (h *upgradeHandler) checkLogReadyCondition(upgrade *harvesterv1.Upgrade) (*harvesterv1.Upgrade, error) {
+	upgradeConfig, err := settings.DecodeConfig[settings.UpgradeConfig](settings.UpgradeConfigSet.Get())
+	if err != nil {
+		return upgrade, err
+	}
+	timeout := upgradeConfig.LogReadyTimeout
```

Comment:
add a range check again, if out of range, use the default value

for e.g. the webhook was removed/defected, an out of range value was injected

Reply:
Done.